### ISSUE TITLE
[BUG] [Formatter] Self-closing tag should end on new line

### ISSUE THREAD

**Issue by nklsw** [djlint/djLint#1234]
- [X] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [X] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [X] I've read the [docs](https://djlint.com)

## System Info

- OS: MacOS 15.3.2
- Python Version 3.12.1
- djLint Version 1.36.4

## Issue

Closing of self-closing tag (`/>`) should be on a new line if the attrs of the tag is split to multiline:

Input:
```html
<input id="{{ field.id }}" type="text" name="{{ field.name }}" class="input example"/>
```

Expected Output:
```html
<input id="{{ field.id }}"
       type="text"
       name="{{ field.name }}"
       class="input example"
/>
```

Actual Output:
```html
<input id="{{ field.id }}"
       type="text"
       name="{{ field.name }}"
       class="input example" />
```

This [example in docs](https://www.djlint.com/docs/best-practices/#format_attribute_template_tags-and-spaceless-conditional-attributes) also shows the behavior I would expect. Am I missing some config or is this a bug?

## How To Reproduce

[Demo Playground](https://www.djlint.com/demo/)

Input
```html
<input id="{{ field.id }}" type="text" name="{{ field.name }}" class="input example"/>
```

**Comment by monosans** [comment]
This now exists as `--single-attribute-per-line`.
